server/controllers/statistics/getStatistics.ts

```diff
@@ -22,6 +24,8 @@ const getStatistics = async (req: Request, res: Response, next:NextFunction) =>
             [sequelize.fn('SUM', sequelize.col('clothes')), 'clothes']],
         }),
       ]);
+    const DONORS = await Donor.count();
+    const CAMPAIGNS = await Campaign.count();
```

Comment:
Add these in the PromiseAll, and rename DONORS, CAMPAIGNS, FOODS, MONEY, and CLOTHES without capital, and send all your data in data